app/utils/importFile.js

```diff
@@ -16,26 +16,18 @@ const importFile = async ({
   documentId,
   collectionId,
 }: Options): Promise<Document> => {
-  return new Promise(async (resolve, reject) => {
-    // non plain text support
-    if (documents.importFiletypes.includes(file.type)) {
-      try {
-        const document = new Document(
-          {
-            parentDocumentId: documentId,
-            collectionId,
-            title: file.name.replace(/\.[^/.]+$/, ""),
-          },
-          documents
-        );
-
-        resolve(await documents.import(document, { publish: true, file }));
-      } catch (err) {
-        reject(err);
-      }
-      return;
-    }
+  // non plain text support
+  if (documents.importFiletypesServer.includes(file.type)) {
+    return await documents.import(
+      { publish: true, file },
+      collectionId,
+      file.name.replace(/\.[^/.]+$/, ""),
+      null,
+      documentId
+    );
```

Comment:
Why have a `text` argument if we're just going to pass null? Suggest:

```suggestion
    const title = file.name.replace(/\.[^/.]+$/, "");

    return await documents.import(
      {
        title,
        documentId,
        collectionId,
      },
      {
        publish: true,
        file
      }
    );
```

Reply:
https://github.com/outline/outline/pull/1469/commits/796fb1a563cd7c14cb1bdbb95ea7521912ad4ea2